docs/src/manual/models.md

```diff
@@ -247,6 +247,21 @@ julia> time_limit_sec(model)
 Inf
 ```
 
+To avoid making unit-conversion mistakes with [`set_time_limit_sec`](@ref),
```

Comment:
I'd frame it more positively, e.g., if your time limit is encoded as a `Dates.Period` object, you can use the following code to convert it to `Float64` for `set_time_limit_sec`.